s3/src/it/scala/monix/connect/s3/S3Suite.scala

```diff
@@ -366,6 +366,62 @@ class S3Suite
     count shouldBe n
   }
 
+  it should "return with latest" in {
```

Comment:
would be good if we could add a test where we try to list more objects than the ones that actually exist? 
like trying to list 10 objects when there under the list prefix there is only 5 or 0 for example

Reply:
added